site/components/pages/Pledge/PledgeDashboard/Profile/index.tsx

```diff
@@ -32,11 +31,21 @@ export const Profile: FC<Props> = (props) => {
     const setProfile = async () => {
       const kns = await getKNSProfile({
         address: props.pledge.ownerAddress,
-        providerUrl: getInfuraUrlPolygon(),
+        providerUrl: getInfuraUrl({
+          chain: "polygon",
+          infuraId: process.env.INFURA_ID!,
```

Comment:
With the "!" you are sure that the env exists, right? (non-null assertion)

But how should the env var locally be defined?
Do all developers now need to have their own `env.local` file in "site" and "app"?

I would remove the "!" if possible 
and extend the readme with something like "you need an env.local file with an infura ID" or alike
and ensure that the function fails without the env var.

What do you think?

Reply:
Yes, good point. We would need to all have an infura id in app and site. I created an env.example file in app and updated the one in site. Do you think I should add this to the contribution / style guide or the main readme?

I replaced the `!` with `as string`. Is that what you were thinking of instead of the `!`?

Thanks!

Reply:
Yes, I would add it to the readme.
Otherwise running site and app locally fails, correct?

In this PR it was added to the README already:
https://github.com/KlimaDAO/klimadao/pull/834/files

Just check if this is already enough to mention it.